src/installer/tests/Assets/TestUtils/TestProjects.targets

```diff
@@ -3,29 +3,22 @@
     This file is imported by the test projects from the artifacts dir or the src/installer/tests dir. It
     provides basic info needed for restore and build with the vanilla SDK.
   -->
-
-  <!--
-    TODO: Remove when the SDK understands the net7.0 tfm.
-  -->
-  <PropertyGroup>
-    <NETCoreAppMaximumVersion>7.0</NETCoreAppMaximumVersion>
-  </PropertyGroup>
-
   <ItemGroup>
-    <KnownFrameworkReference Include="Microsoft.NETCore.App"
+    <!-- Uncomment on change to a new SDK that doesn't support the new TFM, and point to the needed version of the runtime. -->
+    <!-- <KnownFrameworkReference Include="Microsoft.NETCore.App"
                              TargetFramework="net7.0"
                              RuntimeFrameworkName="Microsoft.NETCore.App"
-                             DefaultRuntimeFrameworkVersion="6.0.3"
-                             LatestRuntimeFrameworkVersion="6.0.3"
+                             DefaultRuntimeFrameworkVersion="7.0.0-preview.4.22201.3"
+                             LatestRuntimeFrameworkVersion="7.0.0-preview.4.22201.3"
                              TargetingPackName="Microsoft.NETCore.App.Ref"
-                             TargetingPackVersion="6.0.3"
+                             TargetingPackVersion="7.0.0-preview.4.22201.3"
                              RuntimePackNamePatterns="Microsoft.NETCore.App.Runtime.**RID**"
                              RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;linux-musl-arm;osx-arm64;maccatalyst-x64;maccatalyst-arm64" />
 
     <KnownAppHostPack Include="Microsoft.NETCore.App"
                       TargetFramework="net7.0"
                       AppHostPackNamePattern="Microsoft.NETCore.App.Host.**RID**"
-                      AppHostPackVersion="6.0.3"
-                      AppHostRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;linux-musl-arm;osx-arm64" />
+                      AppHostPackVersion="7.0.0-preview.4.22201.3"
+                      AppHostRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;linux-musl-arm;osx-arm64" /> -->
   </ItemGroup>
```

Comment:
Please remove all the commented out code. Whenever we need to bring this back (for .NET8) we will look into history anyway.